ToolTip.ios.js

```diff
@@ -7,50 +7,50 @@ var {
   NativeModules,
   findNodeHandle,
 } = require('react-native');
+var PropTypes = require('prop-types');
```

Comment:
can you add this to package.json?

Also it would be awesome to have those vars as const/let.